tests/lib/prepare-restore.sh

```diff
@@ -566,11 +566,22 @@ prepare_suite_each() {
             ausearch -i -m AVC --checkpoint "$RUNTIME_STATE_PATH/audit-stamp" || true
             ;;
     esac
+
+    # shellcheck source=tests/lib/pkgdb.sh
+    . "$TESTSLIB"/pkgdb.sh
+    distro_list_packages "$RUNTIME_STATE_PATH/package-selection"
 }
 
 restore_suite_each() {
     rm -f "$RUNTIME_STATE_PATH/audit-stamp"
 
+    if [ -e "$RUNTIME_STATE_PATH/package-selection" ]; then
```

Comment:
Can you please look at https://github.com/snapcore/snapd/pull/7168/files#diff-d2dd0567bc81c6005b7c0f2e727f9b57R531 for some inspiration as to where this should be placed. Ideally we'd roll that under `testbed-tool`. I can work with you on that.

Reply:
We already store a bunch of data under RUNTIME_STATE. The testbed-tool introduces yet another location, this time it's `/var/lib/testbed-tool` which is actually quite hard to track down in the code. I feel we should pick one place and use it everywhere rather than putting it in random places for each tool.